product_docs/docs/biganimal/release/reference/cli.mdx

```diff
@@ -55,6 +55,12 @@ press [Enter] to continue in the web browser...
 Credential "ba-user1" created!
 ```
 
+If the access token for a BigAnimal user has expired before it is stored in a local credential, reset the credential to create a new access token. 
```

Comment:
```
If the access token for a BigAnimal user has expired before it is stored in a local credential
```
I'm not sure what this means? may you elaborate. The access token is stored in credential since the cred is created. And it is get refreshed only if the refresh token hasn't expired. 
`reset-credential` is only needed when the refresh-token expired.

Reply:
Hi Zane, Am changing the description to below:

If a credential you created before has expired, reset the credential using the following command. You must re-authenticate yourself to BigAnimal.

```shell
$ biganimal reset-credential ba-user1 
Visit this URL to login: https://auth.biganimal.com/activate?user_code=****-****
or press [Enter] to continue in the web browser 

Credential "ba-user1" reset operation succeeded

```

Reply:
i do have queries which are as follows:

- What are the conditions/scenarios for a credential to expire?
- i noticed a typo in the messaging. Can we replace 
           Credential "test" is reset operation succeeded 
           with 
           The credential "test" is reset successfully
- @drothery-edb some of the messaging in the doc code block are not the latest. Is it a priority to update these?